src/components/timeslot-editor.ts

```diff
@@ -139,7 +139,12 @@ export class TimeslotEditor extends LitElement {
         if (!!icons) {
           return html`
             <span style="margin-left: auto; margin-right: auto">
-                ${icons.map((icon) => html`<ha-icon icon="${icon}"></ha-icon>`)}
+              ${icons.map(
+                icon =>
+                  html`
+                    <ha-icon icon="${icon}"></ha-icon>
+                  `
+              )}

```

Comment:
The changes to this file are a result of running `npm run format`. (I did not edit this file “manually”.)